File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/save-training-event-participants-xlsx_param.ts


import { TrainingEventParticipantXLSX_Item } from './training-event-participant-xlsx_item';
import { ISaveTrainingEventParticipantsXLSX_Param } from './isave-training-event-participants-xlsx_param';

export class SaveTrainingEventParticipantsXLSX_Param implements ISaveTrainingEventParticipantsXLSX_Param {
  
	public TrainingEventID?: number;
	public ParticipantsExcelStream?: any;
	public Participants?: TrainingEventParticipantXLSX_Item[];
  
}

File: GTTS/INL.TrainingService.Models/SaveTrainingEventParticipantsXLSX_Param.cs
using System;
using System.Collections.Generic;
using System.IO;
using INL.Services.Models;

namespace INL.TrainingService.Models
{
    public class SaveTrainingEventParticipantsXLSX_Param : ISaveTrainingEventParticipantsXLSX_Param
    {
        public Int64? TrainingEventID { get; set; }
        public int? ModifiedByAppUserID { get; set; }
        public Stream ParticipantsExcelStream { get; set; }
        public List<TrainingEventParticipantXLSX_Item> Participants { get; set; }

        public SaveTrainingEventParticipantsXLSX_Param()
        {
            Participants = new List<TrainingEventParticipantXLSX_Item>();
        }
    }
}
